plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouter.sol

```diff
@@ -43,18 +51,36 @@ contract PaymentStandardExitRouter is
         );
 
         challengeStandardExitController = PaymentChallengeStandardExit.Controller(
-            _framework, _spendingConditionRegistry, STANDARD_EXIT_BOND
+            _framework, _spendingConditionRegistry
         );
 
         processStandardExitController = PaymentProcessStandardExit.Controller(
-            _framework, _ethVault, _erc20Vault, STANDARD_EXIT_BOND
+            _framework, _ethVault, _erc20Vault
         );
+
+        bond = BondSize.buildParams(INITIAL_BOND_SIZE);
     }
 
     function standardExits(uint192 _exitId) public view returns (PaymentExitDataModel.StandardExit memory) {
         return standardExitMap.exits[_exitId];
     }
 
+    /**
+     * @notice Gets the standard exit bond size.
+     */
+    function bondSize() public view returns (uint256) {
```

Comment:
`StandardExit` is already in the contract name though, is it necessary to repeat it?
It won't clash with IFE and piggyback bonds because it's scoped to this contract.

Having said that, IFE and piggyback bonds _will_ conflict with each other, so maybe it's more consistent to explicitly use `startStandardExitBondSize` here as you're suggesting...

Reply:
oh, ah...so this probably comes form the stupid/complex multi-inheritence style I introduced to glue things together while making code seperate. I am guessing our client would only get the `PaymentExitGame` contract and not casting to this contract. So when they are using the contract, they will eventually see `bond`, `startIFEBond`, `piggybackBond` instead?

Reply:
Ah yeah - so it will be clearer if we make it `startStandardExitBond`, `startIFEBond`, `piggybackBond`